recipes/yoga/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+class YogaConan(ConanFile):
+    name = "yoga"
+    description = "cross-platform layout engine which implements Flexbox"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://yogalayout.com/"
+    topics = ("layout engine", "flexbox")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
```

Comment:
If this is a static library and the shared option is disabled in the validate method, there's no reason to define it at all iirc, so feel free to drop it :)